api.planx.uk/inviteToPay/createPaymentSendEvents.ts

```diff
@@ -14,9 +17,9 @@ enum Destination {
 }
 
 interface CombinedResponse {
```

Comment:
This type is repeated in `api.planx.uk/send/createSendEvents.ts` but I'm ignoring this for now as this PR is already bigger than it should be.